src/openrct2-ui/scripting/ScGraphicsContext.hpp

```diff
@@ -42,7 +42,9 @@ namespace OpenRCT2::Scripting
             dukglue_register_property(
                 ctx, &ScGraphicsContext::secondaryColour_get, &ScGraphicsContext::secondaryColour_set, "secondaryColour");
             dukglue_register_property(
-                ctx, &ScGraphicsContext::ternaryColour_get, &ScGraphicsContext::ternaryColour_set, "ternaryColour");
+                ctx, &ScGraphicsContext::tertiaryColour_get, &ScGraphicsContext::tertiaryColour_set, "ternaryColour");
```

Comment:
So this keeps support for `ternaryColour` around, shouldn't we mark this as deprecated, to remove it later?

Reply:
FWIW, I have created https://github.com/OpenRCT2/OpenRCT2/issues/16834 .